fast_and_reliable.py

```diff
@@ -0,0 +1,30 @@
+import time
+
+
+def average_runtime(date_structure):
+    """
+    The function checks 1000 times how many times the word zwitterion appears in the data structure.
+    :param date_structure: List or set of words.
+    :return: Run time of the function.
+    """
+    start = time.time()
```

Comment:
I know it will probably be a minor and negligible, but you should start to measure time after defining counter_word and loop_number.